src/common/SearchEmptyState.js

```diff
@@ -0,0 +1,55 @@
+/* @flow */
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { ZulipButton, Label } from './';
+
+const styles = StyleSheet.create({
```

Comment:
Inspired by [Consistency: rename 'moreStyles' to 'componentStyles'](https://github.com/zulip/zulip-mobile/commit/4d03ba310ddd62f1ae37c4f4643d7492b2a10539)